### DISCUSSION TITLE
q群

### DISCUSSION THREAD

**Discussion by M21212** [shanmiteko/LotteryAutoScript#173] [General]
q群还在嘛  有些东西还不是很懂。

**Comment by shanmiteko** [comment]
已暂时开放

**Reply by M21212** [reply]
大佬，群号多少，明天加

**Reply by shanmiteko** [reply]
软件运行的时候可以看到

**Reply by ojbkk** [reply]
> 软件运行的时候可以看到

你好，这个群搜索不到

**Reply by shanmiteko** [reply]
> > 软件运行的时候可以看到
> 
> 你好，这个群搜索不到

群处于半开放状态，搜索限制已关

**Reply by ojbkk** [reply]
> > > 软件运行的时候可以看到
> > 
> > 你好，这个群搜索不到
> 
> 群处于半开放状态，搜索限制已关

大佬那咋进呀🌝

**Reply by shanmiteko** [reply]
> > > > 软件运行的时候可以看到
> > > 
> > > 你好，这个群搜索不到
> > 
> > 群处于半开放状态，搜索限制已关
> 
> 大佬那咋进呀🌝

已开

**Reply by ojbkk** [reply]
> > > > > 软件运行的时候可以看到
> > > > 
> > > > 你好，这个群搜索不到
> > > 
> > > 群处于半开放状态，搜索限制已关
> > 
> > 大佬那咋进呀🌝
> 
> 已开

OK谢谢，已进
